src/resources/Search/SearchInterfaces.ts

```diff
@@ -0,0 +1,40 @@
+import {RestUserIdType} from '../Enums';
+
+export interface RestTokenParams {
+    userIds: RestUserId[];
+    userDisplayName: string;
+    userGroups: string[];
+    canSeeUserProfileOf: string[];
+    pipeline: string;
+    filter: string;
+    searchHub: string;
+    salesforceOrganizationId: string;
+    validFor: number;
+    salesforceUser: string;
+    salesforceCommunityUrl: string;
+    salesforceFallbackToAdmin?: boolean;
+    usertype?: string; // this is really usertype and not userType
+    licenseDefinitionKey?: string;
+    superUserToken: string;
+    commerce?: RestCommerceParameters;
+    scope: number; // Deprecated: This property is exposed for backward compatibility reasons only.
+}
+
+export interface RestUserId {
+    name: string;
+    type: RestUserIdType;
+    provider: string;
+    infos: {};
```

Comment:
I'm not sure if this empty object will cause type errors later. I see in other resources, we use normally `Record` or `any` instead.

Reply:
Since this can be filled with any object. do you think I could use `Object` type ?

Reply:
I'd use `any` in this case IMO. You could check this https://github.com/coveo/platform-client/blob/master/src/resources/MachineLearning/ModelInformation/ModelInformationInterfaces.ts We've use `any` for several models in ML info, because they might be filled with any object. Somebody has to do something with the swagger! :joy: